src/components/card/card.tsx

```diff
@@ -0,0 +1,63 @@
+import { useBuilder } from "src/context-providers/builder";
+import { ICardProps } from "./types";
+import {
+    CardBody,
+    CardDragger,
+    CardIcon,
+    CardContent,
+    CardActions,
+    CardAction,
+    CardDetails,
+} from "./card.styles";
+import { BinIcon } from "@lifesg/react-icons/bin";
+import { CopyIcon } from "@lifesg/react-icons/copy";
+import { EnvelopeIcon } from "@lifesg/react-icons/envelope";
+import { EFieldType } from "src/schemas/types";
+import { Text } from "@lifesg/react-design-system/text";
+
+export const Card = ({ field }: ICardProps) => {
+    const { getFocusControls } = useBuilder();
+    const { isFocused, setFocus } = getFocusControls(field.internalId);    
```

Comment:
I'd like this to be coming in as a prop instead as I hope to reuse the component in the element listing in the side panel